fair/run.py

```diff
@@ -55,9 +55,9 @@
         "exec": "pwsh -command \". '{0}'\"",
         "extension": "ps1"
     },
-    "powershell": {
```

Comment:
I don't think this should be removed

Reply:
It wasn't in the dictionary originally, I added it to the dictionary an then changed it to batch in a later commit. It can be added back in but it will need testing to ensure the environmental variables work and script executes correctly.